models/articles.js

```diff
@@ -0,0 +1,42 @@
+const db = require("../db/connection");
+
+async function getArticlesWithCommentCount() {
+  const result = await db.query(
+    `
+    SELECT articles.*, COUNT(comments.comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC
+  `
+  );
+  return result.rows.map((article) => ({
+    ...article,
+    comment_count: Number(article.comment_count),
+    created_at: new Date(article.created_at),
+  }));
+
+}
+
+async function getArticleById(id) {
```

Comment:
Dont need a comment_count in this trello ticket, please remove this functionality